Formula/composer-switcher.rb

```diff
@@ -0,0 +1,18 @@
+class ComposerSwitcher < Formula
+  desc "Composer Mirror Repository Switcher"
+  homepage "https://www.github.com/persiliao/composer-switcher"
+  url "https://github.com/persiliao/composer-switcher/archive/v0.1.0.tar.gz"
+  version "0.1.0"
+  sha256 "8a2911dbdad2f6dfc036b6e55a5c79bc57aa1b35c6a5736ef6780b9f599e701a"
+
+  depends_on "composer"
```

Comment:
It looks like we don't need bottles for this formula

```suggestion
  sha256 "8a2911dbdad2f6dfc036b6e55a5c79bc57aa1b35c6a5736ef6780b9f599e701a"

  bottle :unneeded

  depends_on "composer"
```